ractor/src/serialization.rs

```diff
@@ -24,7 +24,8 @@ pub trait BytesConvertable {
 }
 
 #[cfg(feature = "blanket_serde")]
-mod impls{
+/// Contains a blanket implementation for all types that implement serde::Serialize and serde::Deserialize
+pub mod impls{
```

Comment:
So a user needs to manually

```rust
use ractor::serialization::impls::*;
```

in order to get the implementations? I feel like this should be a `pub use impls::*` somewhere so they get the blanket implementations for free.

Reply:
No, apparently this module doesn't even have to be public for the blanket implementation to be available, for some reason I vaguely remember not receiving the blanket implementation without making the module public, but can't seem to reproduce it now.
Activating the "cluster" and "blanket_serde" feature is enough on its own to receive the blanket implementations on all the types.